src/mlpack/methods/bayesian_ridge/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Define the files we need to compile
+# Anything not in this list will not be compiled into the output library
+set(SOURCES
+  bayesian_ridge.hpp
+  bayesian_ridge_impl.hpp
+  bayesian_ridge.cpp
+)
+
+# add directory name to sources
+set(DIR_SRCS)
+foreach(file ${SOURCES})
+  set(DIR_SRCS ${DIR_SRCS} ${CMAKE_CURRENT_SOURCE_DIR}/${file})
+endforeach()
+# append sources (with directory name) to list of all mlpack sources (used at the parent scope)
+set(MLPACK_SRCS ${MLPACK_SRCS} ${DIR_SRCS} PARENT_SCOPE)
+
+add_cli_executable(bayesian_ridge)
+add_python_binding(bayesian_ridge)
```

Comment:
We should add a a main test as well, https://github.com/mlpack/mlpack/tree/master/src/mlpack/tests/main_tests does have a bunch of examples, that we could use as a basis.